components/Chat/ChatInput.vue

```diff
@@ -347,9 +357,41 @@
         }
       },
 
-      // Defer some load events til user interacts with chat
-      async onFocus () {
-        await this.updateEmoteList();
+      updateEmoteList() {
+        this.updateEmoteListInStore();
+
+        if( this.emoteList && this.emoteLinkMap && this.emoteLinkMap.size === this.emoteList.length ) {
+          console.log( 'Skipping emoteList update.' );
+          return;
+        }
+
+        if( !this.emoteLinkMap ) {
```

Comment:
This may need to be moved into the store later, I can see a map of emote to url being helpful elsewhere, but this is great as is. Just noting it might be useful elsewhere, but I'm not sure where.

Reply:
Maybe with issues #134 #128 #116 #6